object.go

```diff
@@ -0,0 +1,1131 @@
+	Category              Category     `graphql:"category" json:"category"`                           // The category that the check belongs to.
+	Description           string       `graphql:"description" json:"description"`                     // Description of the check type's purpose.
+	EnableOn              iso8601.Time `graphql:"enableOn" json:"enableOn"`                           // The date when the check will be automatically enabled.
+	Enabled               bool         `graphql:"enabled" json:"enabled"`                             // If the check is enabled or not.
+	Filter                Filter       `graphql:"filter" json:"filter"`                               // The filter that the check belongs to.
+	Id                    ID           `graphql:"id" json:"id"`                                       // The id of the check.
+	Level                 Level        `graphql:"level" json:"level"`                                 // The level that the check belongs to.
+	Name                  string       `graphql:"name" json:"name"`                                   // The display name of the check.
+	Notes                 string       `graphql:"notes" json:"notes"`                                 // Additional information about the check.
+	Owner                 CheckOwner   `graphql:"owner" json:"owner"`                                 // The owner of the check.
+	Type                  CheckType    `graphql:"type" json:"type"`                                   // The type of check.
+	UpdateFrequency       string       `graphql:"updateFrequency" json:"updateFrequency"`             // The minimum frequency of the updates.
+	UpdateRequiresComment string       `graphql:"updateRequiresComment" json:"updateRequiresComment"` // Whether the check requires a comment or not.
+	Url                   string       `graphql:"url" json:"url"`                                     // The url to the check.
+}
+
+// ManualCheckFrequency represents .
+type ManualCheckFrequency struct {
+	FrequencyTimeScale FrequencyTimeScale `graphql:"frequencyTimeScale" json:"frequencyTimeScale"` // The time scale type for the frequency.
+	FrequencyValue     int                `graphql:"frequencyValue" json:"frequencyValue"`         // The value to be used together with the frequency scale.
+	StartingDate       iso8601.Time       `graphql:"startingDate" json:"startingDate"`             // The date that the check will start to evaluate.
+}
+
+// NewRelicIntegration represents .
+type NewRelicIntegration struct {
+	AccountKey                  string       `graphql:"accountKey" json:"accountKey"`                                   // The New Relic Account key for this integration.
+	AllowManualSyncAlertSources string       `graphql:"allowManualSyncAlertSources" json:"allowManualSyncAlertSources"` // Indicates if manual alert source synchronization can be triggered.
```

Comment:
this one should also be a bool - this makes me a little scared about the generator is really not doing the right thing.  Can we figure out why?